frontend/appflowy_flutter/packages/appflowy_ui/lib/src/theme/data/appflowy_default/semantic.dart

```diff
@@ -240,7 +240,7 @@ class AppFlowyDefaultTheme implements AppFlowyThemeBuilder {
     );
 
     final fillColorScheme = AppFlowyFillColorScheme(
-      primary: AppFlowyPrimitiveTokens.neutral100,
+      primary: AppFlowyPrimitiveTokens.neutral900,
```

Comment:
Are these token changes generated by the script?

Reply:
This is a temporary adjustment, which has already been discussed with the UI designer